src/pages/Explorer/Explorer.tsx

```diff
@@ -3,10 +3,73 @@ import { Helmet } from 'react-helmet-async';
 import { Grid } from '@material-ui/core';
 
 import Summary from '@components/Summary/Summary';
+import Table, { HeaderType, RowsProps } from '@components/Table/Table';
+
+import axios from '../../utils/axios/axios';

```

Comment:
Remember to use absolute paths instead relative ones, so:
`@utils/axios/axios`

Reply:
Good catch, changed.